dlp/dlp_inspect_image_file.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+
+# [START dlp_inspect_text_file]
+# Import the Google Cloud Data Loss Prevention library
+import google.cloud.dlp
+
+
+def inspect_image_file(project_id='YOUR_PROJECT_ID',
+                       filepath='path/to/image.png'):
```

Comment:
No hanging indents, please.

If you really wanna be clever run it through `black`.

Reply:
Surprisingly, `black` wanted me to just have a long line that violates the 79 characters rule, so I didn't follow it. Fixed indentation in https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1756/commits/a8d248c9c3471ce9281ec6cecae2494b9cbce40c.